scripts/update/Updater.php

```diff
@@ -298,5 +300,21 @@ public function update($initialVersion) {
         }
 
         $this->skip('6.9.0', '6.15.0');
+
+        if ($this->isVersion('6.15.0')){
+            $userFactory = new UserFactoryService([
+                UserFactoryService::OPTION_CLASS_USER => \core_kernel_users_GenerisUser::class
+            ]);
+            $this->getServiceManager()->register(UserFactoryService::SERVICE_ID, $userFactory);
+
+            /** @var common_ext_ExtensionsManager $extensionManager */
+            $extensionManager = $this->getServiceManager()->get(common_ext_ExtensionsManager::SERVICE_ID);
+            $config = $extensionManager ->getExtensionById('generis')->getConfig('auth');
```

Comment:
Should be added to adapter or only `AuthAdapter` ?

Reply:
Yes, changed.